integration/tests/iam_v1_to_v2_force_upgrade.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#shellcheck disable=SC2034
+test_name="iam_v1_force_upgrade_to_v2"
+test_upgrades=true
+test_upgrade_strategy="none"
+test_deploy_inspec_profiles=(a2-iam-v1-integration)
+test_upgrade_inspec_profiles=(a2-iam-legacy-integration a2-api-integration)
+# Note: we can't run diagnostics AND inspec, so skip diagnostics
+test_skip_diagnostics=true
+
+# on this version, Automate had upgrade-to-v2 and the first three v2 data migrations
+OLD_VERSION=20190501153509
+OLD_MANIFEST_DIR="${A2_ROOT_DIR}/components/automate-deployment/testdata/old_manifests/"
+DEEP_UPGRADE_PATH="${OLD_MANIFEST_DIR}/${OLD_VERSION}.json"
+
+
+do_deploy() {
```

Comment:
this test script:
1. deploys an older version of Automate on IAM v1
2. runs `a2-iam-v1-integration` inspec tests
3. upgrades Automate to the latest build. This force-upgrades the system to IAM v2
4. runs `a2-iam-legacy-integration` and  `a2-api-integration` inspec tests

Reply:
I like your PR comments here and on the next file. Suggest moving those into the script itself as it will help readers looking at the raw file.

Reply:
I'm not sure what you mean by "raw file". I can see all the comments in the raw file view: https://raw.githubusercontent.com/chef/automate/9309dc4db427889fc025064bfa14639a35aef997/integration/tests/iam_v1_to_v2_force_upgrade.sh